modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1911,6 +1911,16 @@ return view.extend({
 				]).then(this.renderContents.bind(this));
 			};
 
+			if (bss.ssid == null) {
+				name = s2.option(form.Value, 'ssid', _('WiFi SSID'), _('SSID must be manually set when a Hidden WiFi is used.'));
```

Comment:
Please change to:
`_('Network SSID')` and `_('The correct SSID must be manually specified when joining a hidden wireless network')`